pages/calculator/[slug].js

```diff
@@ -54,8 +54,20 @@ export default function CalculatorSlugRoute({ page, calculatorConfig }) {
 
   const isFirstPage = () => page.slug === 'head-initial-1-cont';
 
+  const url = "https://clearviction.org/"
```

Comment:
This URL is for the homepage, but it should be for the first page of the calculator, `/calculator/head-initial-1-cont`, and let's also make the variable more descriptive, so perhaps something like `calcFirstPageUrl`.

Reply:
So is the reason why we are doing this altogether because we would like to avoid confusion in the code? What I mean is that the href before was using check another conviction which would lead to https://clearviction.org/calculator/head-initial-1-cont just like I am doing right now.

Reply:
Yes the reason is because it feels a bit confusing to have the a button that is not the check another conviction button using a url called check another conviction.

Reply:
Completely makes sense